src/MatrixClient.ts

```diff
@@ -837,6 +838,15 @@ export class MatrixClient extends EventEmitter {
         });
     }
 
+    /**
+     * Gets information about a particular user.
+     * @param {string} userId the user ID to lookup
+     * @returns {Promise<IAdminWhois>} resolves to the whois information
+     */
+    public getUserWhois(userId: string): Promise<IAdminWhois> {
```

Comment:
should probably be shoved to an admin APIs class similar to how unstableApis works